app/src/pages/[resourceType]/[id].tsx

```diff
@@ -79,41 +121,66 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
-          <Tabs.List>
-            <Tabs.Tab value="json">JSON</Tabs.Tab>
-            {decodedElm != null && <Tabs.Tab value="elm">ELM</Tabs.Tab>}
-            {decodedCql != null && <Tabs.Tab value="cql">CQL</Tabs.Tab>}
-            {jsonData.text && <Tabs.Tab value="narrative">Narrative</Tabs.Tab>}
-          </Tabs.List>
-          <Tabs.Panel value="json" pt="xs">
-            <Prism language="json" colorScheme="light">
-              {JSON.stringify(jsonData, null, 2)}
-            </Prism>
-          </Tabs.Panel>
-          {decodedCql != null && (
-            <Tabs.Panel value="cql" pt="xs">
-              {/* eslint-disable  @typescript-eslint/no-explicit-any */}
-              <Prism language={'cql' as any} colorScheme="light">
-                {/* eslint-enable  @typescript-eslint/no-explicit-any */}
-                {decodedCql}
-              </Prism>
-            </Tabs.Panel>
-          )}
-          {decodedElm != null && (
-            <Tabs.Panel value="elm" pt="xs">
+        <div>
+          <Tabs variant="outline" defaultValue="json">
+            <Tabs.List>
+              <Tabs.Tab value="json">JSON</Tabs.Tab>
+              {decodedElm != null && <Tabs.Tab value="elm">ELM</Tabs.Tab>}
+              {decodedCql != null && <Tabs.Tab value="cql">CQL</Tabs.Tab>}
+              {jsonData.text && <Tabs.Tab value="narrative">Narrative</Tabs.Tab>}
+              {dataRequirements?.Library.resourceType == 'Library' && (
+                <Tabs.Tab value="datarequirements">Data Requirements</Tabs.Tab>
+              )}
+            </Tabs.List>
+            <Tabs.Panel value="json" pt="xs">
               <Prism language="json" colorScheme="light">
-                {decodedElm}
+                {JSON.stringify(jsonData, null, 2)}
               </Prism>
             </Tabs.Panel>
-          )}
-          {jsonData.text && (
-            <Tabs.Panel value="narrative">
-              <Space h="sm" />
-              {parse(jsonData.text.div)}
-            </Tabs.Panel>
-          )}
-        </Tabs>
+            {decodedCql != null && (
+              <Tabs.Panel value="cql" pt="xs">
+                {/* eslint-disable  @typescript-eslint/no-explicit-any */}
+                <Prism language={'cql' as any} colorScheme="light">
+                  {/* eslint-enable  @typescript-eslint/no-explicit-any */}
+                  {decodedCql}
+                </Prism>
+              </Tabs.Panel>
+            )}
+            {decodedElm != null && (
+              <Tabs.Panel value="elm" pt="xs">
+                <Prism language="json" colorScheme="light">
+                  {decodedElm}
+                </Prism>
+              </Tabs.Panel>
+            )}
+            {jsonData.text && (
+              <Tabs.Panel value="narrative">
+                <Space h="sm" />
+                {parse(jsonData.text.div)}
+              </Tabs.Panel>
+            )}
+            {dataRequirements?.Library.resourceType == 'Library' && (
```

Comment:
If you make the change below, this will change to just be `dataRequirements?.resourceType === 'Library'` I believe. Also, it's good practice to use strict equality (`===`). You can read more about that [here](https://www.tektutorialshub.com/typescript/strict-equality-loose-equality-in-typescript/).